tanner/utils/mysql_db_helper.py

```diff
@@ -18,7 +18,7 @@ def __init__(self):
                                       )
         return conn
 
-    async def check_db_exists(self, db_name, ):
+    async def check_db_exists(self, db_name):
         conn = await self.connect_to_db()
```

Comment:
If it is not necessary, please don't do it.

Reply:
What is the use of this?

Reply:
I'm not sure of it use either 😅

Reply:
I think its a typo.

Reply:
Alright.